modules/host.go

```diff
@@ -111,14 +119,33 @@ type (
 	// StorageObligation contains information about a storage obligation that
 	// the host has accepted.
 	StorageObligation struct {
+		ObligationId             types.FileContractID `json:"obligationid"`
+		FileSize                 uint64               `json:"filesize"`
```

Comment:
FileSize is the wrong name here, it's the data size. A contract houses many partial files, not a single file.